database/migrations/20191216192400_create-profile_info-table.js

```diff
@@ -0,0 +1,31 @@
+
+exports.up = async function(knex, Promise) {
+    await knex.schema.createTable('profile_info', tbl => {
+        tbl
+            .increments();
+        tbl
+            .string('profile_pic')
+            .notNullable();
+        tbl
+            .string('first_name')
+            .notNullable();
+        tbl
+            .string('last_name')
+            .notNullable();
+        tbl
+            .string('bio', 500)
+            .notNullable();
+        tbl
+            .integer('user_id')
+            .unsigned()
+            .notNullable()
+            .references('id')
+            .inTable('users')
```

Comment:
Minor aesthetic quibble: I'd move this to line 22. So: `.references('id').inTable('users')`.